cfssl.go

```diff
@@ -67,6 +67,10 @@ var Config struct {
 	domain            string
 	ip                string
 	remote            string
+	short             bool
+	list              bool
+	family            string
```

Comment:
Essentially you should probably only use one or the other. Using both takes the intersection. How does this sound?
```
  -family="": regular expression selecting specific scan families to run (defaults to all)
  -scanner="": regular expression selecting specific scanners to run (defaults to all)
```